packages/fast-components-styles-msft/src/dialog/index.ts

```diff
@@ -1,51 +1,43 @@
-import { DesignSystem, withDesignSystemDefaults } from "../design-system";
-import { ComponentStyles, ComponentStyleSheet } from "@microsoft/fast-jss-manager";
+import { DesignSystem } from "../design-system";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
 import { DialogClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
-import { applyAcrylicMaterial } from "../utilities/acrylic";
 import { applyElevatedCornerRadius } from "../utilities/border";
+import { backgroundColor } from "../utilities/design-system";
 import { applyElevation, ElevationMultiplier } from "../utilities/elevation";
 
-/* tslint:disable-next-line */
-const styles: ComponentStyles<DialogClassNameContract, DesignSystem> = (
-    config: DesignSystem
-): ComponentStyleSheet<DialogClassNameContract, DesignSystem> => {
-    const designSystem: DesignSystem = withDesignSystemDefaults(config);
-    const backgroundColor: string = designSystem.backgroundColor;
-
-    return {
-        dialog: {
-            display: "none",
-            '&[aria-hidden="false"]': {
-                display: "block",
-            },
-        },
-        dialog_positioningRegion: {
-            display: "flex",
-            justifyContent: "center",
-            position: "fixed",
-            top: "0",
-            bottom: "0",
-            left: "0",
-            right: "0",
-            overflow: "auto",
-        },
-        dialog_modalOverlay: {
-            position: "fixed",
-            top: "0",
-            left: "0",
-            right: "0",
-            bottom: "0",
-            ...applyAcrylicMaterial(backgroundColor, 0.6, 0.9, true),
-        },
-        dialog_contentRegion: {
-            marginTop: "auto",
-            marginBottom: "auto",
-            background: backgroundColor,
-            ...applyElevatedCornerRadius(),
-            ...applyElevation(ElevationMultiplier.e14),
-            zIndex: "1",
+const styles: ComponentStyles<DialogClassNameContract, DesignSystem> = {
+    dialog: {
+        display: "none",
+        '&[aria-hidden="false"]': {
+            display: "block",
         },
-    };
+    },
+    dialog_positioningRegion: {
+        display: "flex",
+        justifyContent: "center",
+        position: "fixed",
+        top: "0",
+        bottom: "0",
+        left: "0",
+        right: "0",
+        overflow: "auto",
+    },
+    dialog_modalOverlay: {
+        position: "fixed",
+        top: "0",
+        left: "0",
+        right: "0",
+        bottom: "0",
+        background: "rgba(0, 0, 0, 0.3)",
```

Comment:
@bheston this should *never* theme whatsoever?

Reply:
Right, always darker, which the transparency gives us. So if the neutral palette was red, it would still look and feel correct.